Mathlib/Data/Rel.lean

```diff
@@ -263,9 +263,21 @@ lemma image_union : image R (s₁ ∪ s₂) = image R s₁ ∪ image R s₂ := b
 
 @[deprecated (since := "2025-07-06")] alias preimage_eq_codom_of_domain_subset := image_union
 
+variable (R) in
+lemma image_iUnion (s : ι → Set α) : image R (⋃ i, s i) = ⋃ i, image R (s i) := by aesop
```

Comment:
Can these be `simp`
```suggestion
@[simp] lemma image_iUnion (s : ι → Set α) : image R (⋃ i, s i) = ⋃ i, image R (s i) := by aesop
```
analogous to [map_iSup](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Hom/CompleteLattice.html#map_iSup).

Reply:
This wouldn't be confluent with `SetRel.mem_image`. Also, `SetRel.image_union` and `Set.image_iUnion` aren't `simp` either.